app/react/src/server/config/webpack.config.prod.js

```diff
@@ -29,7 +29,7 @@ export default function() {
           screw_ie8: true,
           warnings: false,
         },
-        mangle: false,
+        mangle: true,
```

Comment:
It's not about debugging, it's about the prod deployment of as storybook being broken. I don't think enabling `mangle` is safe and it would mean that in Dev mode everything might be fine but the storybook deploy could be broken.